SpotifyAPI.Example/LocalControl.cs

```diff
@@ -70,8 +85,8 @@ public void UpdateInfos()
             versionLabel.Text = status.Version.ToString();
             repeatShuffleLabel.Text = status.Repeat + @" and " + status.Shuffle;
 
-            if (status.Track != null) //Update track infos
-                UpdateTrack(status.Track);
+            //if (status.Track != null) //Update track infos
+            //    UpdateTrack(status.Track);
```

Comment:
[SpotifyLocalAPI.cs#103](https://github.com/JohnnyCrazy/SpotifyAPI-NET/pull/140/files#diff-8aafce16036626e8c5113f4188ed4e2eR103) this line ignores old status response if it is raising event for the first time. So we can get track information through OnTrackChange event when application started. Those lines became unnecessary.